Library/Formula/ppl.rb

```diff
@@ -0,0 +1,25 @@
+class Ppl < Formula
+  desc "Parma Polyhedra Library"
+  homepage "http://bugseng.com/products/ppl"
+  url "ftp://ftp.cs.unipr.it/pub/ppl/releases/1.1/ppl-1.1.tar.xz"
+  version "1.1"
```

Comment:
URL is simple enough that `brew` can understand the version. Can scrap the manual `version` line.